email/1.4/modules/ROOT/pages/email-troubleshooting.adoc

```diff
@@ -0,0 +1,196 @@
++
+[source,plain-text]
+----
+DEBUG 2021-04-26 18:51:08,536 [_pollingSource_imapattachmentFlow/executor.01] [processor: ; event: ] org.mule.extension.email.internal.mailbox.BaseMailboxPollingSource: Poll will be skipped, since last poll emails are still being processed
+INFO  2021-04-26 18:51:09,166 [[MuleRuntime].uber.04: [imapattachment].imapattachmentFlow.CPU_LITE @111719e0] [processor: imapattachmentFlow/processors/1/processors/2; event: 820f7fe0-a6d9-11eb-a84b-147dda4dba09] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: "" as Binary {base: "64"}
+DEBUG 2021-04-26 18:51:15,360 [_pollingSource_imapattachmentFlow/executor.01] [processor: ; event: ] org.mule.extension.email.internal.mailbox.BaseMailboxPollingSource: Email [172] was not processed.
+....
+----
+
+
+== Troubleshoot SMTPS and Gmail Connection Issues
+
+The method that you use to troubleshoot SMTPS connection issues depends on whether or not your Gmail account uses two-factor authentication:
+
+=== Two-Factor Authentication
+
+If your account uses two-factor authentication value generate an app-specific password and use that instead of your normal password.
+See https://support.google.com/accounts/answer/185833[Sign in Using App Password] for details. You do not need to enable *Less Secure Apps* in your Gmail account.
+
+=== Password Based Authentication
+
+If your Gmail account does not use two-factor authentication, set up and enable *Less Secure Apps* in your Gmail account, and if your password does not work, go to https://accounts.google.com/b/0/DisplayUnlockCaptcha[Allow Access to Your Google Account] and follow these steps:
+
+. Enter your username and password.
+. Enter the letters on the captcha screen.
+. Return to your Mule app and rerun the flow.
+
+
+== Understand the Behavior of the Different Protocols
+
+If your problem is about the behavior of each protocol, check the RFC documents. A Request for Comments (RFC) is a publication from the Internet Society (ISOC) and its associated bodies, most prominently the Internet Engineering Task Force (IETF), the principal technical development and standards-setting bodies for the Internet. The IETF adopts some of the proposals published as RFCs as Internet Standards.
+Some documents that you can check are:
+
+* https://tools.ietf.org/html/rfc5322[RFC-5322 - Internet Message Format]
+* https://tools.ietf.org/html/rfc1064[RFC-1064 - IMAP2 - INTERACTIVE MAIL ACCESS PROTOCOL - VERSION 2]
+* https://tools.ietf.org/html/rfc1939[RFC-1939 - POP3 - Post Office Protocol - Version 3]
+* https://tools.ietf.org/html/rfc5321[RFC-5321 - SMTP - Simple Mail Transfer Protocol]
+* https://tools.ietf.org/html/rfc2045[RFC-2045 - MIME - Multipurpose Internet Mail Extensions Part One: Format of Internet Message Bodies]
+
+
+== Understand Common Throws
+
+Here is a list of common throw messages and how to interpret them.
+
+* EMAIL:EMAIL_NOT_FOUND
+
+  The email with the emailId provided can not be found in a mailbox folder.
```

Comment:
If you like, you can use the following format for each of the messages:
```suggestion
* EMAIL:EMAIL_NOT_FOUND: The email identified by `emailId` cannot be found in a mailbox folder.
```
If you do not mean to refer to a syntax element, remember that ID is always allcaps and separate: email ID.